modules/launcher/src/main/java/org/mule/runtime/module/launcher/util/ReflectionUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.launcher.util;
+
+import static org.mule.runtime.core.util.Preconditions.checkArgument;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Provides utilities to work with reflection.
+ */
+public class ReflectionUtils
```

Comment:
Use ClassUtils from core. Perhaps renaming ClassUtils to ReflectionUtils if it makes sense but doesn't have to be in this PR.